libraries/SPI/examples/Example2_MoreSPIPorts/Example2_MoreSPIPorts.ino

```diff
@@ -0,0 +1,54 @@
+/* Author: Owen Lyke
+  Created: May 13 2019
+  License: MIT. See SparkFun Arduino Apollo3 Project for more information
+
+  This example demonstrates how to use Arduino SPI
+*/
+
+#include "SPI.h"
+
+#define CS_PIN 2
+
+#define SPI_SPEED 1000000
+#define SPI_ORDER MSBFIRST
+#define SPI_MODE SPI_MODE0
+
+SPISettings mySettings(SPI_SPEED, SPI_ORDER, SPI_MODE);
+
+const char *msg = "Hello world!";
+
+//SPIClass SPI(); //This is default and automatically defined on RedBoard/ATP/Nano. Uses pads 5/6/7 (SCK/MISO/MOSI).
```

Comment:
Just a nit: you are correctly using the pad vs pin notation but perhaps a user would not understand that pads 5/6/7 are actually (13, 12, 11) on the RedBoard (w/ Uno pin naming scheme). 

Should there be a note explaining this in this example?